controllers/kokumetricsconfig_controller.go

```diff
@@ -470,8 +470,13 @@ func uploadFiles(r *KokuMetricsConfigReconciler, authConfig *crhchttp.AuthConfig
 			return err
 		}
 		ingressURL := kmCfg.Status.APIURL + kmCfg.Status.Upload.IngressAPIPath
-		uploadStatus, uploadTime, err := crhchttp.Upload(authConfig, contentType, "POST", ingressURL, body)
+		uploadInfo, exists := fileInfo[file]
```

Comment:
```suggestion
		uploadInfo, exists := fileInfo[file]
		if !exists {
			<log something>
			continue // skip processing this file
		}
```

This `exists` should be immediately checked. If the file isn't there, then I'm pretty sure `uploadInfo` will be `nil` and the logic below will fail.

Reply:
Added a check for exists immediately below grabbing the file info